packages/documentation/copy/en/reference/Type Compatibility.md

```diff
@@ -70,6 +70,14 @@ greet(dog); // OK
 Note that `dog` has an extra `owner` property, but this does not create an error.
 Only members of the target type (`Pet` in this case) are considered when checking for compatibility.
 
+Be aware, however, that [object literals](https://www.typescriptlang.org/docs/handbook/2/objects.html#excess-property-checks)
+may only specify known properties. For example, because we have explicitly specified that `dog` is
+of type `Pet`, the following code is invalid:
+
+```ts
+let dog: Pet = { name: "Lassie", owner: "Rudd Weatherwax" }; // Error
+```
+
 This comparison process proceeds recursively, exploring the type of each member and sub-member.
```

Comment:
I would move this sentence back up above your newer text, i.e. add your stuff to the bottom of this section.

Reply:
Good point. I'd go further and say it's properly part of the preceding paragraph (Note that dog...), so I'm going to merge them.